Client/Nimbus/NimbusSearchBarLayer.swift

```diff
@@ -8,7 +8,7 @@ final class NimbusSearchBarLayer {
 
     // MARK: - Public methods
     public func getDefaultPosition(from nimbus: FxNimbus = FxNimbus.shared) -> SearchBarPosition {
-        let isAtBottom = nimbus.features.search.value().awesomeBar.position.isBottom
+        let isAtBottom = nimbus.features.awesomeBar.value().position.isBottom
```

Comment:
I think the `position.isBottom` and `isPositionFeatureEnabled` needs addressing, but this PR is about large scale re-arranging rather than fine grain detail.

Reply:
Sure, we can totally do that, at a later point. But we have to be careful as this is going to be under experiment, so we shouldn't change it until the experiment is over.